static/app/components/modals/addDashboardWidgetModal.tsx

```diff
@@ -115,23 +116,29 @@ type State = {
   selectedDashboard?: SelectValue<string>;
 };
 
-const newDiscoverQuery = {
+const newDiscoverQuery: WidgetQuery = {
   name: '',
   fields: ['count()'],
+  columns: [],
+  aggregates: ['count()'],
```

Comment:
should we add these changes to the widgetBuilder too?

Reply:
I'm going to do that today!